packages/contracts-ts/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './constants'
```

Comment:
Should we be cherry-picking constants so we don't accidentally expose un-released values?

Reply:
agree with @refcell  I don't dislike being more explicit on what we exports in here, also being this the main entry file, otherwise we just proxying constants.ts and not sure about the need of this file?

Reply:
Good point actually there is no need for this file to exist